item.rb

```diff
@@ -0,0 +1,32 @@
+class Item
+  attr_accessor :publish_date, :genre, :label, :author, :id
+
+  def initialize(publish_date, archived: true)
+    @id = Random.rand(1..1000)
+    @publish_date = publish_date
+    @archived = archived
+  end
+
+  def add_genre(genre)
+    @genre = genre.name
+    genre.items.push(self) unless genre.items.include?(self)
+  end
+
+  def add_author(author)
+    @author = author.first_name
+    author.items.push(self) unless author.items.include?(self)
+  end
+
+  def add_label(label)
+    @label = label.title
+    label.items << self unless label.items.include?(self)
+  end
+
+  def can_be_archived?
+    @publish_date < 10.year.ago
+  end
```

Comment:
- [ ] To correctly check that the published date is less than ten years old, we should use the `date` library, I mean `require 'date'` and add a couple of lines to this method 👇🏽 

```ruby
today = Date.today
ten_years_ago = Date.new(today.year - 10, today.month, today.day)
```